src/components/Mission.js

```diff
@@ -0,0 +1,33 @@
+// import { useDispatch } from 'react-redux';
+
+const Mission = (prop) => {
+  const { mission } = prop;
+//   const dispatch = useDispatch();
+
```

Comment:
[Optional] It's good practice to remove a commented code on pull request to be merged to master branch. I hope you will remove this one ✌